packages/drivers/routerlicious-host/src/urlResolver.ts

```diff
@@ -58,6 +58,7 @@ export class ContainerUrlResolver implements IUrlResolver {
         const fluidResolvedUrl = resolvedUrl as IFluidResolvedUrl;
 
         const parsedUrl = parse(fluidResolvedUrl.url);
+        // eslint-disable-next-line no-null/no-null
         assert(parsedUrl.pathname !== null, 0x0b7 /* "Pathname should be defined" */);
```

Comment:
Seems like it should also check undefined?

Reply:
I used a revert of one of Tyler's commits to fix a bunch of no-nulls in bulk, this was one of them.  At least per the node type definitions, the url module will always return a string or null here so an undefined check shouldn't be necessary.  Best option would be to drop usage of `parse()` since it's deprecated (here and elsewhere through the repo).

Reply:
Looks like I Frankensteined the eslint file wrong though, so these no-nulls might have to come back out.  I'll see what the options are.